README.md

```diff
@@ -104,7 +104,7 @@ Travis CI is used to validate romcal builds to ensure functionality is working a
 
 ## Builds <a name="builds"></a>
 
-Running `npm run build` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
+Running `npm install && npm run build` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
```

Comment:
> `npm install` will already automatically execute the `npm run build`.
> `npm run build` is only necessary if you modify code, and you want then to update the frontend/browser library.

When I tested this with `master` after merging #147, it did not executed it automatically (it throwed errors about missing dependencies). And the `master` branch was clean.

Reply:
<details>
  <summary>
    <code>npm run build</code>
  </summary>

  ```bash
  > romcal@1.3.0-rc2 build /home/ts/git/c10n/romcal
  > npm run build:babel && npm run build:webpack


  > romcal@1.3.0-rc2 build:babel /home/ts/git/c10n/romcal
  > babel src --out-dir dist

  sh: babel: command not found
  npm ERR! file sh
  npm ERR! code ELIFECYCLE
  npm ERR! errno ENOENT
  npm ERR! syscall spawn
  npm ERR! romcal@1.3.0-rc2 build:babel: `babel src --out-dir dist`
  npm ERR! spawn ENOENT
  npm ERR! 
  npm ERR! Failed at the romcal@1.3.0-rc2 build:babel script.
  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
  npm WARN Local package.json exists, but node_modules missing, did you mean to install?

  npm ERR! A complete log of this run can be found in:
  npm ERR!     /home/ts/.npm/_logs/2020-01-13T17_18_14_480Z-debug.log
  npm ERR! code ELIFECYCLE
  npm ERR! errno 1
  npm ERR! romcal@1.3.0-rc2 build: `npm run build:babel && npm run build:webpack`
  npm ERR! Exit status 1
  npm ERR! 
  npm ERR! Failed at the romcal@1.3.0-rc2 build script.
  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
  npm WARN Local package.json exists, but node_modules missing, did you mean to install?

  npm ERR! A complete log of this run can be found in:
  npm ERR!     /home/ts/.npm/_logs/2020-01-13T17_18_14_501Z-debug.log
  ```
</details>

<details>
  <summary>
    <code>/home/ts/.npm/_logs/2020-01-13T17_18_14_501Z-debug.log</code>
  </summary>

  ```bash
  0 info it worked if it ends with ok
  1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'build' ]
  2 info using npm@6.9.0
  3 info using node@v10.16.3
  4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
  5 info lifecycle romcal@1.3.0-rc2~prebuild: romcal@1.3.0-rc2
  6 info lifecycle romcal@1.3.0-rc2~build: romcal@1.3.0-rc2
  7 verbose lifecycle romcal@1.3.0-rc2~build: unsafe-perm in lifecycle true
  8 verbose lifecycle romcal@1.3.0-rc2~build: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/ts/git/c10n/romcal/node_modules/.bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/home/ts/perl5/bin:/usr/share/Modules/bin:/home/ts/perl5/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux:/home/ts/.rvm/bin:/usr/local/texlive/2019/bin/x86_64-linux
  9 verbose lifecycle romcal@1.3.0-rc2~build: CWD: /home/ts/git/c10n/romcal
  10 silly lifecycle romcal@1.3.0-rc2~build: Args: [ '-c', 'npm run build:babel && npm run build:webpack' ]
  11 silly lifecycle romcal@1.3.0-rc2~build: Returned: code: 1  signal: null
  12 info lifecycle romcal@1.3.0-rc2~build: Failed to exec build script
  13 verbose stack Error: romcal@1.3.0-rc2 build: `npm run build:babel && npm run build:webpack`
  13 verbose stack Exit status 1
  13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
  13 verbose stack     at EventEmitter.emit (events.js:198:13)
  13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
  13 verbose stack     at ChildProcess.emit (events.js:198:13)
  13 verbose stack     at maybeClose (internal/child_process.js:982:16)
  13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
  14 verbose pkgid romcal@1.3.0-rc2
  15 verbose cwd /home/ts/git/c10n/romcal
  16 verbose Linux 4.18.0-151.el8.x86_64
  17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "build"
  18 verbose node v10.16.3
  19 verbose npm  v6.9.0
  20 error code ELIFECYCLE
  21 error errno 1
  22 error romcal@1.3.0-rc2 build: `npm run build:babel && npm run build:webpack`
  22 error Exit status 1
  23 error Failed at the romcal@1.3.0-rc2 build script.
  23 error This is probably not a problem with npm. There is likely additional logging output above.
  24 verbose exit [ 1, true ]
  ```
</details>

Reply:
"When I tested this" --> when you tested what exactly?

I can confirm you that `npm install` perform automatically an `npm run build`, check the travis builds for example. It's also working when npm publishing a new version. This is not new from the last release, it have been added since the beta3.

Reply:
I have tested twice:
- right after you merged #147 (I have no logs);
- a few minutes ago on 2211220 commit (current `master`) and the result was same.

I have posted the relevant logs in my [previous comment](https://github.com/romcal/romcal/pull/146#discussion_r365927547) on commit 2211220 (current `master`).

Reply:
And I should also note that I always had to run `npm install`, even before you started maintaining romcal. And I currently run CentOS 8 Stream x86_64.

Reply:
> I can confirm you that `npm install` perform automatically an `npm run build`

Well, if that is so, the current `README.md` is still wrong. Current [`README.md`](https://github.com/romcal/romcal#builds-) states that one should run `npm run build`, not `npm install`.

And till now, I have never checked if `npm install` also runs `npm run build`. I have just checked that out and indeed you are right that `npm install` also builds romcal. I’ll fix it.

Reply:
you should only have to run `npm install` when the `package.json` have been changed.
Otherwise, you might have an issue with your node/npm installation.

Reply:
Good to know. :smiley:

Anything else you want me to change in this PR? :smiley: